clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/view/EcsClusterProvider.java

```diff
@@ -36,4 +51,52 @@ public EcsClusterProvider(Cache cacheView) {
   public Collection<EcsCluster> getAllEcsClusters() {
     return ecsClusterCacheClient.getAll();
   }
+
+  // TODO include[] input of Describe Cluster is not part of this implementation, need to implement
+  // in the future.
+  public Collection<Cluster> getEcsClusterDescriptions(String account, String region) {
+    Collection<Cluster> clusters = new ArrayList<>();
+    List<String> filteredEcsClusters =
+        ecsClusterCacheClient.getAll().stream()
+            .filter(
+                cluster ->
+                    account.equals(cluster.getAccount()) && region.equals(cluster.getRegion()))
+            .map(cluster -> cluster.getName())
+            .collect(Collectors.toList());
+    List<List<String>> batchClusterList =
```

Comment:
Not a blocker, but could you add some debug logging here? It would be great to be able to enable debugging and be able to determine the size of the batchClusterList (how many partitions did we split the original list into?)